cmd/verify.go

```diff
@@ -34,7 +34,7 @@ import (
 	"github.com/redhat-certification/chart-verifier/pkg/chartverifier"
 )
 
-var Version = "1.0.1"
+var Version = "1.1.0"
```

Comment:
Should this be a build property instead of being hardcoded here. For now OK just want to know if there is a plan to be more dynamic with this version.

Reply:
This was made a var so it could be changed as part of a build.

go build -ldflags="-X 'github.com/redhat-certification/chart-verifier/cmd.Version=1.1.0'"

But no support yet for it to be dynamic.